### ISSUE TITLE
How do you install/use this?

### ISSUE THREAD

**Issue by swimnbird** [alladinian/Tuna#1]
I'm a bit of a newbie but I'm not used to how this is set up, im not even able to run the packages in the Github Zip. Any guidance or a video would be greatly appreciated.

**Comment by alladinian** [comment]
Hi there :)

Thanks for showing interest in my project and for reaching out!

Tuna is distributed as a Swift package and can be easily integrated into any modern iOS / macOS project.

Here are some simple integration instructions:

https://developer.apple.com/documentation/xcode/adding_package_dependencies_to_your_app

I have also attached a small project you can use as a reference (see the project settings & view controller).

I hope that this makes sense.

[TunaSample.zip](https://github.com/alladinian/Tuna/files/6643468/TunaSample.zip)

**Comment by swimnbird** [comment]
Thank you so much.. sorry for asking twice. I could have learned patience:)

**Comment by swimnbird** [comment]
I thought I asked this before but had trouble locating where it was. Sorry about that.

I recall trying the sample before and I believe I got an error similar to what I'm getting now:
`git@github.com:alladinian/Tuna.git: An unknown error occurred. failed to connect to github.com: Operation timed out (-1)`

Apparently this is an error associated with certain versions of xCode (for anyone else observing this post at a later date)



So instead I installed the dependencies into a new project manually and use the code. It seems to work (but doesnt do anything except print some info via terminal from the 'doSoemthing' Function) but am not sure how to go about creating a pitch detector on (I'm on MacOS). I'm referencing [Beethoven's Guitar Tuner Example](https://github.com/vadymmarkov/Beethoven/tree/master/Example/GuitarTuner) 
is this the best available project to reference or is there possibly a better one?
